### ISSUE TITLE
Add term enlarged head of the humerus

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10372]
**Preferred term label:**
Enlarged humeral head

**Synonyms**
Large humeral head

**Definition (free text, please give PubMed ID)**
Enlarged head of the humerus

**Parent term (use hpo.jax.org/app)**
Abnormal humeral head morphology HP:0003887

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778 (PMID: 12624140)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000884
- label: Enlarged humeral head
- parents: [Abnormal humeral head morphology (HP:0003887)](https://hpo.jax.org/app/browse/term/HP:0003887)
- pmid: 
- def: Size (volume) of the head of the humerus above the upper limit of normal.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10372
- synonyms:
